app/Http/Controllers/PageController.php

```diff
@@ -266,11 +266,13 @@ public function showDelete(string $bookSlug, string $pageSlug)
         $page = $this->pageRepo->getBySlug($bookSlug, $pageSlug);
         $this->checkOwnablePermission('page-delete', $page);
         $this->setPageTitle(trans('entities.pages_delete_named', ['pageName' => $page->getShortName()]));
+        $times_used_as_template = Book::where('default_template', '=', $page->id)->count();
```

Comment:
Variable name does not follow project conventions, but use may be redundant (lang file comments).